ProcessLib/BoundaryCondition/BoundaryCondition.cpp

```diff
@@ -49,6 +51,15 @@ BoundaryConditionBuilder::createBoundaryCondition(
                     integration_order, shapefunction_order, parameters);
     }
 
+    //
+    // Special boundary conditions
+    //
+    if (type == "Pressure")
```

Comment:
``Pressure`` is a general name, and this sort of BC is only for normal stress or normal traction BC. Therefore, I think it would be good to use ``NormalTraction`` or ``NormalStress`` for this specific BC type.